docs/source/guide/tutorials/advanced/self_sl.rst

```diff
@@ -0,0 +1,226 @@
+############################
+Use Self-Supervised Learning
+############################
+
+This tutorial introduces how to train a model using self-supervised learning and fine-tune the model with pre-trained weights.
+OpenVINO™ Training Extensions provides self-supervised learning methods for :doc:`multi-classification <../../explanation/algorithms/classification/multi_class_classification>` and :doc:`semantic segmentation <../../explanation/algorithms/segmentation/semantic_segmentation>`.
+
+The process has been tested on the following configuration.
+
+- Ubuntu 20.04
+- NVIDIA GeForce RTX 3090
+- Intel(R) Core(TM) i9-10980XE
+- CUDA Toolkit 11.7
+
+.. note::
+
+  This example demonstates how to work with :doc:`self-supervised learning for classification <../../explanation/algorithms/classification/multi_class_classification>` and :doc:`self-supervised learning for semantic segmentation <../../explanation/algorithms/segmentation/semantic_segmentation>`.
+
+*************************
+Setup virtual environment
+*************************
+
+To create a universal virtual environment for OpenVINO™ Training Extensions, please follow the installation process in the :doc:`quick start guide <../../get_started/quick_start_guide/installation>`.
+
+***************************
+Multi-class classification
+***************************
+.. _tutorial_selfsl_classification:
+
+.. note::
+
+  To prepare `flowers dataset <https://www.tensorflow.org/hub/tutorials/image_feature_vector#the_flowers_dataset>`_, refer to :doc:`how to train classification models <../base/how_to_train/classification>`.
+
+------------
+Pre-training
+------------
+
+1. We can check which models are provided for this task like the command below. All details are in :doc:`how to train classification models <../base/how_to_train/classification>`. We will choose :ref:`MobileNet-V3-large-1x <classificaiton_models>` like :doc:`how to train classification models <../base/how_to_train/classification>`.
```

Comment:
There are to many refers to classification tutorial. Also, we can omit "otx find --task classification" since it is advanced tutorial . I perfer to just mention here that we can use the same templates and backbones as for original classification for Self-SL task. For that tutorial we will use MobileNet-V3-large-1x as we did in classification tutorial. 
I would rather delete `otx find ` example

Reply:
Done